app/routes/my-projects/project/recommendations/add.js

```diff
@@ -19,6 +19,7 @@ export default class MyProjectsProjectRecommendationsAddRoute extends Route {
   async setupController(controller, model) {
     super.setupController(controller, model);
     controller.set('dispositions', model.dispositions);
+    controller.set('allActions', model.dispositions.length <= 1 ? true : null);
```

Comment:
So, I think these should be computeds on the controller or a component — for example, what if the # of dispositions change? the value of `allActions` will be out of sync.

Reply:
ohhh. Hmm. I don't think the number of dispositions would ever change within one user's browsing session though. I thought it's predetermined by the CRM.  I guess if somehow a planner added more actions via CRM while a user was using the website, it could happen between page loads... hmmm